package.json

```diff
@@ -24,7 +24,6 @@
     "ember-cli-inject-live-reload": "^1.3.1",
     "ember-cli-qunit": "^1.0.0",
     "ember-cli-release": "0.2.3",
-    "ember-disable-prototype-extensions": "^1.0.0",
```

Comment:
Since the array observers are removed can we disable prototype extensions again?